event:
  - id: 10
    name: 'E-STOP'
    animation:
      both:
        animation_id: 0
        duration: 3
        color: [255,255,255]
        range: [0, 47]

  - id: 25
    name: 'TURN-LEFT'
    animation:
      front:
        animation_id: 0
        duration: 1
        wait_after_sequence: 0.5
        repeat: 2
        keep_state: True
        color: 0x0F0AFF
        range: [0, 47]
      tail:
        animation_id: 0
        duration: 3
        keep_state: False
        color: [0,255,0]
        range: [45, 3]

  - id: 20
    name: 'TURN-LEFaT'
    interrupting: True
    animation:
      both:
        animation_id: 0
        duration: 2
        color: [255,0,255]
        range: [0, 47]

  - id: 2137
    name: 'SLIDE-FIVE'
    animation:
      both:
        animation_id: 0
        duration: 1
        wait_after_sequence: 0
        repeat: 2
        keep_state: True
        color: [255,0,255]
        range: [0, 47]